iron-iconset.html

```diff
@@ -296,8 +296,12 @@
       // and having iconMap map names to indices, then and index is just
       // iconMap[identifier] || identifier (be careful of zero, store indices
       // as 1-based)
-      return this.iconMap[identifier] ||
+      var offset = this.iconMap[identifier] ||
              this.iconMap[this.iconNames[Number(identifier)]];
+      if (!offset) {
+        console.error('Icon with identifier "' + identifier + '" not found! Did you specify an iconset?');
```

Comment:
Some of our larger users are very anxious about usage `console` logging in random components. I think there is a `this._error` you can use instead?